spec/hardware/disk.fmf

```diff
@@ -9,20 +9,29 @@ description: |
            # Number or string, the amount of storage requested.
            # Bytes are assumed when no unit is specified.
          - size: 1234|">= 512 GiB"
+           # String, disk model requested.
+         - model-name: "well-known disk model"
 
```

Comment:
`versionchanged` should be here, see e.g. https://github.com/teemtee/tmt/blob/main/spec/hardware/network.fmf#L27:

```
    .. versionchanged:: 1.32
       Added `model-name` into specification.
```